src/main/resources/config/application-dev.yml

```diff
@@ -87,8 +88,8 @@ server:
 # ===================================================================
 managementportal:
     common:
-        baseUrl: http://localhost:9000 # Modify according to your server's URL
-        managementPortalBaseUrl: http://localhost:8080/
+        baseUrl: http://localhost:8081 # Modify according to your server's URL
```

Comment:
changes here do not seem to affect anything, but it was a mismatch between the angular dev server port after changes in #725 so I set it to match again

Reply:
indeed, just used for building mail and metatokens